java/code/src/com/redhat/rhn/common/localization/LocalizationService.java

```diff
@@ -647,7 +647,7 @@ public boolean hasMessage(String messageId) {
         List<String> tmp = new LinkedList<String>();
 
         // Get locales of installed documentations
-        File f = new File("/srv/www/htdocs/docs");
+        File f = new File(Config.get().getString("documentroot") + "/docs");
```

Comment:
I think we should have one of two options:

- pass a default value, to not force it to be defined on the configuration files
- Add a wrapper method to ConfigDefaults to return the document root, and that method deals with the defaults.